src/state/apis/foxy/foxyApi.ts

```diff
@@ -55,7 +55,7 @@ export type FoxyOpportunity = {
   tokenAssetId: AssetId
   rewardTokenAssetId: AssetId
   pricePerShare: string
-  withdrawInfo: WithdrawInfo
+  withdrawInfo: Record<AccountId, WithdrawInfo>
```

Comment:
We can have as many `withdrawInfo`s as we have `AccountId`s for each opportunity.